### PR TITLE
Fix a security issue of KeepassXC extension

### PR DESCRIPTION
**Pull request by zkytech**
## Description

1.Replace `echo` with `stdin.write` . #983 
2.Show HUD when copy password or  username 
 
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)

### CODE REVIEW

File: extensions/keepassxc/package.json [hunk 1]
```diff
@@ -38,8 +38,8 @@
       "type": "password",
       "required": true,
       "title": "Password",
-      "description": "Password of Keepass Database",
-      "placeholder": "Please Input Password of Keepass Database"
+      "description": "Password of Keepass Database. For More Information about Security, Please Visit README on Github",
```
Comment (pernielsentikaer) [inline extensions/keepassxc/package.json:41]:
```suggestion
      "description": "Password of Keepass database. For more information about security, please visit README on GitHub",
```

File: extensions/keepassxc/package.json [hunk 2]
```diff
@@ -38,8 +38,8 @@
...
+      "placeholder": "Please Input Password of Keepass Database."
```
Comment (pernielsentikaer) [inline extensions/keepassxc/package.json:42]:
```suggestion
      "placeholder": "Please input password of Keepass database."
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @zkytech 👋 

Thanks for the update! 💎 

I corrected some description/placeholders – could you correct the rest?

I tested the extension, and it still works as expected, awesome 💣 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by zkytech** [comment]
@pernielsentikaer  Done.

**Review by pernielsentikaer — APPROVED** [review]
Awesome @zkytech 🔥 

Thanks for the corrections, approved! 👏

**Comment by sxn** [comment]
🎉

**Pull request merged**